pkg/rpc/server.go

```diff
@@ -274,6 +277,31 @@ Methods:
 	s.WriteResponse(req, w, results)
 }
 
+func (s *Server) getStorage(ps Params) (interface{}, error) {
+	param, ok := ps.Value(0)
+	if !ok {
+		return nil, errInvalidParams
+	}
+
+	scriptHash, err := param.GetUint160FromHex()
+	if err != nil {
+		return nil, errInvalidParams
+	}
+
+	param, ok = ps.Value(1)
+	if !ok {
+		return nil, errInvalidParams
+	}
+
+	key, err := param.GetBytesHex()
+	if err != nil {
+		return nil, errInvalidParams
+	}
+
+	item := s.chain.GetStorageItem(scriptHash.Reverse(), key)
+	return item, nil
```

Comment:
Is it JSONized correctly or are we missing something here?

Reply:
Yes, we are. Fixed.
Query:
```json
{
    "jsonrpc": "2.0",
    "method": "getstorage",
    "params": [
        "67a5086bac196b67d5fd20745b0dc9db4d2930ed",
        "8eb60cf0dd78bb881d8bc1ccdc24e3c4aa5576de"
    ],
    "id": 1
}
```
Result:
```json
{
    "jsonrpc": "2.0",
    "id": 1,
    "result": "002e597611"
}
```